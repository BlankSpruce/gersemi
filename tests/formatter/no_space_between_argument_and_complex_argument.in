if(NOT(-1 EQUAL (${v})))
endif()

if(NOT(-1 EQUAL (${v}))AND(1 EQUAL 1))
endif()

if(BAR(-1))
endif()

if("BAR"(-1))
endif()

if([[BAR]](-1))
endif()

if((-1)BAR)
endif()

if((-1)"BAR")
endif()

if((-1)[[BAR]])
endif()

if(BAR(-1)BAZ)
endif()

if(BAR(-1)"BAZ")
endif()

if(BAR(-1)[[BAZ]])
endif()