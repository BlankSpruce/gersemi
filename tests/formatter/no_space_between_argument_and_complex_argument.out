if(NOT (-1 EQUAL (${v})))
endif()

if(NOT (-1 EQUAL (${v})) AND (1 EQUAL 1))
endif()

foo(BAR (-1))

foo("BAR" (-1))

foo([[BAR]] (-1))

foo((-1) BAR)

foo((-1) "BAR")

foo((-1) [[BAR]])

foo(BAR (-1) BAZ)

foo(BAR (-1) "BAZ")

foo(BAR (-1) [[BAZ]])
